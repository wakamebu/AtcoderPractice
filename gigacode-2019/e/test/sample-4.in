0 1
99991 1
