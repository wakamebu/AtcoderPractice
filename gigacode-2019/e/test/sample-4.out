0.000010000900081007291
